exasol/slc/internal/tasks/export/export_container_base_task.py

```diff
@@ -97,20 +104,24 @@ def _create_export_info(
             depends_on_image=image_info_of_release_image,
             release_goal=str(self.release_goal),
             release_name=str(self.release_name),
-            output_file=str(output_file),
+            output_file=str(output_file) if output_file else None,
         )
         return export_info
 
-    def _get_cache_file_path(self, image_info_of_release_image):
+    def _get_cache_file_path(
+        self,
+        image_info_of_release_image: ImageInfo,
+        export_directory_future: AbstractTaskFuture,
+    ) -> Tuple[Path, str, str]:
         release_image_name = image_info_of_release_image.get_target_complete_name()
-        export_path = Path(self._export_directory_future.get_output()).absolute()
+        export_path = Path(export_directory_future.get_output()).absolute()
```

Comment:
Use function parameter instead of `self._export_directory_future`, because this is already checked against None.